Readme update

rin-st opened the pull request:
Second part from #99 :

> Also probably we can remove the yarn chain, yarn deploy stuff and point it them to SE-2 docs for that

- removed `requirements` and `quickstart` blocks
- renamed `documentation` block to `getting started` and added link to [QuickStart Guide](https://docs.scaffoldeth.io/quick-start) so it replaced both `requirements` and `quickstart` blocks

Also: 
- removed outdated `git clone` info from main description
- some smaller adjustments

First part from #99:
> I think maybe we should have section which points people to creating third party extensions.

I'm not sure about this part (55% not to add it/ 45% to add), because third-party extensions part looks too specific for such a short readme. But I'm very open to change it

Fixes #99

technophile-04 reviewed (commented):
Tysm Rinat 🙌

Since this will be shown on npm readme. I think its important to highlight `npx create-eth@latest` command? 

Maybe we have it like this? 

----
----


## 🏗 create-eth
CLI to create decentralized applications (dapps) using Scaffold-ETH 2.

<h4 align="center">
  <a href="https://docs.scaffoldeth.io">SE-2 Documentation</a> |
  <a href="https://scaffoldeth.io">SE-2 Website</a>
</h4>

## Requirements

Before you begin, you need to install the following tools:

- [Node (>= v18.18)](https://nodejs.org/en/download/)
- Yarn ([v1](https://classic.yarnpkg.com/en/docs/install/) or [v2+](https://yarnpkg.com/getting-started/install))
- [Git](https://git-scm.com/downloads)

## Quickstart

To get started with Scaffold-ETH 2, follow the steps below:

1. Install the latest version of Scaffold-ETH 2

```
npx create-eth@latest
```

This command will install all the necessary packages and dependencies, so it might take a while.

> [!NOTE]
> You can also initialize your project with one of our extensions to add specific features or starter-kits. Learn more in our [extensions documentation](https://docs.scaffoldeth.io/extensions/).


----
----




> I'm not sure about this part (55% not to add it/ 45% to add), because third-party extensions part looks too specific for such a short readme. But I'm very open to change it

Ohh yes yes let's not add that part

rin-st commented:
From the [issue](https://github.com/scaffold-eth/create-eth/issues/99) I thought we don't need requirements and yarn chain/deploy etc but this

> Since this will be shown on npm readme.

makes sense

Updated Readme as you suggested

Thanks Shiv!

Pabl0cks commented on README.md:
```diff
@@ -1,16 +1,9 @@
-> If you find any bug, please report as [issue](https://github.com/scaffold-eth/create-eth/issues) or send a message in [🏗 scaffold-eth developers chat](https://t.me/joinchat/F7nCRK3kI93PoCOk)
-
-# 🏗 create-eth
+## 🏗 create-eth
```
Maybe we can keep the h1?

```suggestion
# 🏗 create-eth
```

rin-st replied:
thanks, fixed here

Pabl0cks reviewed (commented):
Nice job @rin-st and @technophile-04 !! Only added a nitpick.

Even I really love how simple the readme looks like now, I have a couple of questions to discuss:

- Do you think we should add some kind of next steps (maybe could be simplified to a 1 liner to the website section) like we have in SE-2 Readme above [documentation section](https://github.com/scaffold-eth/scaffold-eth-2/?tab=readme-ov-file#documentation)
- Have realized we don't have any reference to BuidlGuidl or any social link. Do you think could be interesting to add a Community section, that could have some of these links:
   - Link to scaffold-eth developers tg chat
   - Link to twitter for project updates (maybe bg + se twitter)
   - Youtube channel for guides / examples, maybe link to playlists
   - Built by buidlguidl? With link to the website

Docs website have all these community links in the footer, but was wondering if it would be handy to have (some of?) them here too. Also the fact that the repo is inside scaffold-eth makes me feel a reference to BuidlGuidl would be nice here.

But just throwing the ideas as they came to my mind, totally fine to ignore and keep it simple 🙏

technophile-04 commented:
> Do you think we should add some kind of next steps (maybe could be simplified to a 1 liner to the website section) like we have in SE-2 Readme above [documentation section](https://github.com/scaffold-eth/scaffold-eth-2/?tab=readme-ov-file#documentation)

Ohh yeah I think one liner is worth it and makes sense having it 🙌

> Have realized we don't have any reference to BuidlGuidl or any social link. Do you think could be interesting to add a Community section, that could have some of these links: 

Ohh yeah maybe instead of having list with pointers maybe in the footer of the README we could have the link like this ?? 

<h4 align="center">
  <a href="https://buidlguidl.com">Buidlguidl</a> |
  <a href="https://scaffoldeth.io">SE-2 Website</a> | 
   <a href="https://scaffoldeth.io">Youtube</a>
</h4>

rin-st commented:
Thanks for the suggestions!

> Do you think we should add some kind of next steps (maybe could be simplified to a 1 liner to the website section) like we have in SE-2 Readme above [documentation section](https://github.com/scaffold-eth/scaffold-eth-2/?tab=readme-ov-file#documentation)

Added it!

> Have realized we don't have any reference to BuidlGuidl or any social link. Do you think could be interesting to add a Community section, that could have some of these links:

Added community section this way for now: 
<img width="1182" alt="image" src="https://github.com/user-attachments/assets/d9e3c675-e1c5-4fe6-a331-753b507189ac" />

Not sure which youtube links we should add. 

Feel free to update the section if needed

technophile-04 reviewed (approved):
Lgtm! Thanks Rinat! 

> Not sure which youtube links we should add.

Umm maybe link to Buidlguidl is doing the job we so don't it probably. 

CC @Pabl0cks once, but this looks great to me!

Pabl0cks commented:
> CC @Pabl0cks once, but this looks great to me!

Yesterday I was trying to get some upgrades but wasn't happy enough to push them. Modified it a bit now, what do you think, is worth to change? Or some change feels forced?

Tried to set a link to youtube and bg website in places that could feel kind of natural, to simplify the footer just with community links.

![image](https://github.com/user-attachments/assets/f93d9959-d6cf-4ebc-8eaf-567702a933cf)

I can push the changes if you feel it enhances current version, if not let's just merge current one! 🙏

rin-st commented:
I like your changes!

Pabl0cks commented:
> I like your changes!

Pushed!

technophile-04 commented:
Thanks all 🙌

Just pushed a couple of commit : 

https://github.com/scaffold-eth/create-eth/pull/186/commits/7dc8a2b7398ca65bde044a2049de65c1c3f828c2 fixed the link, for some reason when I clicked earlier link it gave me this: 
<img width="1118" alt="Screenshot 2025-01-10 at 11 06 55 AM" src="https://github.com/user-attachments/assets/3677339c-7007-4c17-8650-cafba3b2af88" />

https://github.com/scaffold-eth/create-eth/pull/186/commits/4cd63fc2e007307183a732c4588728785b502af0: removed the backlink to SE-2 github repo for contributing, we are moving towards create-eth being the main source of truth soon 🙌
